templates/DataDiscovery/index.html.twig

```diff
@@ -85,6 +85,9 @@
                             <td valign="top">
                                 <input type="button" id="clear-button" class="button" value="Clear/Show All" onclick="clearAll();">
                             </td>
+                            <td valign="top">
+                                <input type="button" id="searchlink-button" class="button teal-button" value="Search V2" onclick="navigateToNewsearch();">
```

Comment:
Still has onClick, use jQuery.on, or if you wanna be cool use: `document.getElementById("searchlink-button").addEventListener("click", navigateToNewsearch);`

Reply:
Can even use an anonymous function in the event listener.